garage/sampler/stateful_pool.py

```diff
@@ -60,7 +60,7 @@ def initialize(self, n_parallel):
             self.manager = mp.Manager()
             self.queue = mp.Queue()
             self.worker_queue = mp.Queue()
-            self.pool = MemmappingPool(
+            self.pool = MemmapingPool(
```

Comment:
Please use `conda env update -n garage` to get the latest joblib, which corrected this typo.

Reply:
Done. Will updated on next push.